nmpc_controller/src/nmpc_controller.cpp

```diff
@@ -213,7 +210,7 @@ NMPCController::NMPCController(ros::NodeHandle &nh, int robot_id) {
   app_->Options()->SetNumericValue("ma57_pre_alloc", 1.5);
   app_->Options()->SetStringValue("fixed_variable_treatment",
                                   "make_parameter_nodual");
-  app_->Options()->SetNumericValue("tol", 1e-3);
+  app_->Options()->SetNumericValue("tol", 5e-3);
```

Comment:
Would this be too big?

Reply:
Ah, this was modified for a different test. Changed.